uberfire-structure/uberfire-structure-backend/src/main/java/org/guvnor/structure/backend/organizationalunit/OrganizationalUnitServiceImpl.java

```diff
@@ -574,23 +515,23 @@ public Boolean isValidGroupId(final String proposedGroupId) {
         return false;
     }
 
-    public void updateRegisteredOUCache(@Observes @org.guvnor.structure.backend.config.Repository SystemRepositoryChangedEvent changedEvent) {
-        registeredOrganizationalUnits.clear();
-        loadOrganizationalUnits();
-    }
-
-    // refresh org unit in case repository changed otherwise it will have outdated information
-    public void updateRegisteredOUonRepoChange(@Observes RepositoryEnvironmentUpdatedEvent changedEvent) {
```

Comment:
did you miss this? and also the updateRegisteredOUCache? Or those are not necessary anymore because there is no cache